### PR TITLE
Make unit tests pass on Windows

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,13 +5,13 @@ buildscript {
...
   dependencies {
-    classpath 'org.gradle.api.plugins:gradle-nexus-plugin:0.7.1'
+    classpath 'com.bmuschko:gradle-nexus-plugin:2.3.1'
   }
 }
 
 apply plugin: 'groovy'
 apply plugin: 'maven'
-apply plugin: 'nexus'
+apply plugin: 'com.bmuschko.nexus'
```
Comment (JakeWharton) [inline build.gradle]: Why is this included in the PR?

Comment (sschuberth) [inline build.gradle]: Hmm, odd. When I first tried, `./gradlew check` did not even start to execute because Gradle could not download `gradle-nexus-plugin-0.7.1`. I just tried again, and now it works. Probably some temporary network hiccup. Anyway version 0.7.1 seems quite outdated and / or the version scheme seems to have changed. [Bintray search](https://bintray.com/search?query=gradle-nexus-plugin) shows `org.gradle.api.plugins:gradle-nexus-plugin:0.3` or `com.bmuschko:gradle-nexus-plugin:2.3.1` to be the latest version (both from Benjamin Muschko), but the latter has a more recent release date. Strangely enough, I do not see version 0.7.1 on Bintray.

I'll update the commit message to not say anymore that it did not work without this change.

Comment (JakeWharton) [inline build.gradle]: Huh. Weird.
